www/js/diary/services.js

```diff
@@ -35,12 +35,12 @@ angular.module('emission.main.diary.services', ['emission.services'])
         'text-align': 'center',
         'background-color': '#33e0bb',
         'color': 'black',
-        'width': '48px',
+        'width': '18%',
```

Comment:
Great to see that this is changed to %
But the colors are still hardcoded here.
Why can't all this be in a CSS style instead of being in javascript?

If it does have to be in javascript, it should be in a separate config module instead of being mixed in with the main functionality.